dockertest/images.py

```diff
@@ -92,6 +92,10 @@ def __init__(self, repo, tag, long_id, created, size,
         self.user = user
 
         self.short_id = long_id[:12]
+        # docker-1.10 output includes a prefix describing the hash type
+        # FIXME: find canonical list
+        if long_id.startswith("sha256:"):
+            self.short_id = long_id[7:19]
```

Comment:
>Why not extend the parsing regex with an (\w+:)? 

That puts the burden of understanding container IDs on the caller. I think it makes more sense for the caller to deal with cid as a black box. But I hate the hardcoded sha256 so I've checked in (unpushed) a different way to do it.